packages/x-charts/src/BarChart/BarLabel/BarLabelPlot.tsx

```diff
@@ -0,0 +1,76 @@
+import * as React from 'react';
+
```

Comment:
Nitpick: is there any reason for these empty lines? 🤔

Reply:
No, I guess my vscode just add them under some circunstances